BraveRewardsUI/Wallet/WalletViewController.swift

```diff
@@ -318,6 +318,26 @@ class WalletViewController: UIViewController, RewardsSummaryProtocol {
   
   // MARK: - Actions
   
+  @objc private func tappedClaimGrantButton(_ sender: ActionButton) {
+    sender.loaderView = LoaderView(size: .small)
+    sender.loaderPlacement = .replacesContent
+    sender.isLoading = true
+    ledgerObserver.grantClaimed = { [weak self] grant in
+      guard let self = self, let grantAmount = BATValue(probi: grant.probi)?.displayString else { return }
+      sender.isLoading = false
+      let claimedVC = GrantClaimedViewController(
+        grantAmount: grantAmount,
+        expirationDate: Date(timeIntervalSince1970: TimeInterval(grant.expiryTime))
+      )
+      self.present(claimedVC, animated: true) {
+        self.tappedNotificationClose()
+      }
+    }
+    if let grant = state.ledger.pendingGrants.first {
+      state.ledger.solveGrantCaptch(withPromotionId: grant.promotionId, solution: "")
```

Comment:
`solveGrantCaptch` typo, 
why is solution an empty string here?

Reply:
Yeah I noticed that, but not worth the fix in brave-core because its going to be renamed anyways with virtual grants PR. And empty string because we don't have captcha, will also be replaced later with a different API call